src/vstest.console/CommandLine/Executor.cs

```diff
@@ -50,21 +52,26 @@ namespace Microsoft.VisualStudio.TestPlatform.CommandLine;
 /// </summary>
 internal class Executor
 {
+    private const string NonARM64RunnerName = "vstest.console.exe";
```

Comment:
Why not simply `runnerName`?

Reply:
because we have different name for runner x64 and arm and so can be confusing if you read only the usage, isn't it?